[DOCS] Updated Older Snippets as per new guidelines.

walter-ind opened the pull request:
### Summary

Updated Older Snippets as per new guidelines.

kibanamachine commented:
Since this is a community submitted pull request, a Jenkins build has not been kicked off automatically. Can an Elastic organization member please verify the contents of this patch and then kick off a build manually?

miukimiu commented:
jenkins test this

miukimiu commented:
jenkins test this

miukimiu commented:
Thanks for opening this PR but according to our [CONTRIBUTING](https://github.com/elastic/eui/blob/master/CONTRIBUTING.md) guidelines:

> Contributors should limit themselves to **three** or less active PRs at any one time, which helps us focus review time towards PRs that are close to a merge event. 

At this moment you have 5 open PRs:
<img width="1013" alt="Screenshot 2020-04-20 at 13 36 42" src="https://user-images.githubusercontent.com/2750668/79752317-067e2d00-830c-11ea-8205-b24b50ebc70f.png">

We'll try to review the PRs but next time try to adhere to our guidelines. 🙂

kibanamachine commented:
Preview documentation changes for this PR: https://eui.elastic.co/pr_3355/

walter-ind commented:
![847](https://user-images.githubusercontent.com/52423075/79753621-d77bb180-8333-11ea-9337-0db9d7920012.png)

Actually, I read in one of the issues stated by one of the community member. So I thought 3 PRs or less than that is true only for community members. However, as per contributing guidelines, it is true for any contributor.

miukimiu commented:
jenkins test this

kibanamachine commented:
Preview documentation changes for this PR: https://eui.elastic.co/pr_3355/

miukimiu commented:
jenkins test this

kibanamachine commented:
Preview documentation changes for this PR: https://eui.elastic.co/pr_3355/

walter-ind commented:
> Thanks @walter-ind,
> 
> I removed the EuiFlyout snippet in favor of #3357. I also fixed a few missing variables.

Sure! I learned more about what to change and how to change. 
Thanks for your guidance!!
